src/components/containers/resetRequest.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable react/no-unused-state */
+import React, { Component } from "react";
+import ResetView from "../views/ResetRequestForm";
+import ResetPasswordAPI from "../../api/resetPasswordAPI";
+
+class ResetRequestPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      emailInputError: "",
+      emailButtonDisable: true
+    };
+  }
+
+  onHandleChange = (event) => {
```

Comment:
can you change the logic in this function and set the email to state if it is valid